app/models/article.rb

```diff
@@ -2,8 +2,9 @@ class Article < ApplicationRecord
   extend FriendlyId
   friendly_id :title, use: :slugged
   belongs_to :section, optional: true
-  has_many :users, through: :authorships
+  has_many :authorships
```

Comment:
Why this?

Reply:
Removal of that extra `has_many :authorships` results in a 500 error:
```
ActiveRecord::HasManyThroughAssociationNotFoundError (Could not find the association :authorships in model Article):
```